scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
The comment above is intended to answer that. It's to be able to ignore net451 build failures on non-windows. 

If we don't do it this way, a net451 failure will fail builds for all tfms.

Anyways, I see your point, the comment could be better. 
I do think better factoring is the more ideal answer to that though.